shared/teams/team/member/add-to-channels.tsx

```diff
@@ -211,14 +231,36 @@ const HeaderRow = ({onCreate, onSelectAll, onSelectNone}) => (
   </Kb.Box2>
 )
 
-const ChannelRow = ({channelname, numMembers, selected, onSelect}) =>
-  Styles.isMobile ? (
+const ChannelRow = ({channelname, conversationIDKey, numMembers, selected, onSelect}) => {
+  const numParticipants = Container.useSelector(
+    s => ChatConstants.getParticipantInfo(s, conversationIDKey).all.length
+  )
+  const activityLevel = 'active' // TODO: plumbing
+  return Styles.isMobile ? (
     <Kb.ClickableBox onClick={onSelect}>
       <Kb.Box2 direction="horizontal" style={styles.item} alignItems="center" fullWidth={true} gap="medium">
-        <Kb.Text type="Body" lineClamp={1} style={Styles.globalStyles.flexOne}>
-          #{channelname}
-        </Kb.Text>
-        <Kb.CheckCircle checked={selected} onCheck={onSelect} />
+        <Kb.Box2 direction="vertical" style={styles.channelContent}>
+          <Kb.Box2
+            direction="horizontal"
+            gap="small"
+            alignSelf="flex-start"
+            style={Styles.globalStyles.flexOne}
+          >
+            <Kb.Text type="Body" lineClamp={1}>
+              #{channelname}
```

Comment:
Seems this can push the check circle off the screen with long channel names
<img width="250" alt="image" src="https://user-images.githubusercontent.com/11968340/76467646-7372de80-63c0-11ea-857b-d86e07ceba1b.png">

Reply:
K I will `...` it.

Reply:
Hmm, I can't seem to get this to work. @buoyad any ideas?

Reply:
Fiddled with it some, pushed a fix